recipes-soletta/soletta/files/add-fbp-passfail-result.patch

```diff
@@ -0,0 +1,34 @@
+Add PASS/FAIL output to test output
+
+---
+diff --git a/tools/run-fbp-tests b/tools/run-fbp-tests
+index 6bb618d..0ebe0e7 100755
+--- a/tools/run-fbp-tests
++++ b/tools/run-fbp-tests
+@@ -241,6 +241,7 @@ def run_tests(args):
+         if curr.should_skip(skips):
+             logging.debug("SKIPPED running %s: %s" % (t, curr.skip_msg))
+             status["skipped"] += 1
++            logging.info("SKIP: %s" % path["file"])
+             continue
+ 
+         if not args.compiled:
+@@ -269,6 +270,7 @@ def run_tests(args):
+                 logging.warning("Dependence test %s for test %s failed. Skipping it",
+                         curr.depends, curr.test_file)
+                 status["skipped"] += 1
++                logging.info("FAIL: %s" % path["file"])
```

Comment:
This skip is caused by the dependence test fail. And this is not same as the specified "skipped cases" by user. 
To calculate the pass rate (PASS/all executable cases) in ptest results, I think it should be looked as FAIL though it is skipped actually. 
And I will update this patch and submit tools/fbp-run-tests patch to soletta.